File: Recursion/sum_of_qube.py
def sum_of_cubes(n):
    if n == 1:
        return 1
    else:
        return n**3 + sum_of_cubes(n - 1)

N = int(input("Enter the value of N: "))
result = sum_of_cubes(N)
print("Sum of cubes:", result)

File: Recursion/sum_of_qube.c
#include <stdio.h>

int sumOfCubes(int n) {
    if (n == 1)
        return 1;
    else
        return (n * n * n) + sumOfCubes(n - 1);
}

int main() {
    int N;
    printf("Enter the value of N: ");
    scanf("%d", &N);
    int result = sumOfCubes(N);
    printf("Sum of cubes: %d\n", result);
    return 0;
}
